src/maze/style.h

```diff
@@ -0,0 +1,25 @@
+#ifndef STYLE_H
+#define STYLE_H
+
+#include <string>
+
+struct StyleConfig {
+    std::string strokeColor;
+    std::string backgroundColor;
+    int strokeWidth;
+};
+
+class Style {
+public:
+    static void init(StyleConfig config);
+    static void destroy();
+
+    static inline const StyleConfig& get() { return instance->config; };
+private: 
+    Style(StyleConfig config);
+    StyleConfig config;
+
+    static Style *instance;
```

Comment:
What is the purpose of the instance variable when you have the config directly in the class?